types/storybook__addon-knobs/index.d.ts

```diff
@@ -52,6 +52,7 @@ export function object<T>(name: string, value: T, groupId?: string): T;
 export function radios<T>(name: string, options: { [s: string]: T }, value?: T, groupId?: string): string;
 
 export function select<T>(name: string, options: { [s: string]: T }, value: T, groupId?: string): T;
+export function select<T extends string>(name: string, options: ReadonlyArray<T>, value: T, groupId?: string): T;
```

Comment:
@alanhchoi 

The following cases should be acceptable.

```
select('padding', [4, 8, 16], 4)
```

`T` must extends string or number.

Reply:
@akameco Hi! That's my bad. Thank you for pointing it out and I created a new pull request https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33796. Please check the test file in the PR and see if it matches with your case.

Reply:
@alanhchoi Thanks you! LGTM 👍